File: MusicPlayer/app/src/main/java/com/example/vladimir/musicplayer/PrefTheme.java
package com.example.vladimir.musicplayer;

import android.content.Context;
import android.content.SharedPreferences;
import android.support.v7.preference.PreferenceManager;

public class PrefTheme {
    public static int getTheme(Context context) {
        SharedPreferences sharedPref = PreferenceManager.getDefaultSharedPreferences(context);
        String currentTheme = sharedPref.getString("theme","def");
        int themeId = R.style.AppTheme;
        switch (currentTheme) {
            case "Yellow":
                themeId = R.style.Yellow;
                break;
            case "Dark":
                themeId = R.style.Dark;
                break;
            case "White":
                themeId = R.style.AppTheme;
                break;
        }

        return themeId;
    }
}

File: MusicPlayer/app/src/main/java/com/example/vladimir/musicplayer/PrefTheme.kt
package com.example.vladimir.musicplayer

import android.content.Context
import android.support.v7.preference.PreferenceManager

object PrefTheme {
    fun getTheme(context: Context): Int {
        val sharedPref = PreferenceManager.getDefaultSharedPreferences(context)
        val currentTheme = sharedPref.getString("theme", "def")
        return when (currentTheme) {
            "Yellow" -> R.style.Yellow
            "Dark" -> R.style.Dark
            else -> R.style.AppTheme
        }
    }
}
